ufersa/plataforma-sabia
# Show technology average rating at technology details page

## Summary

![image](https://user-images.githubusercontent.com/10481028/97904519-6162f900-1d1f-11eb-987c-75d11bae384a.png)

![image](https://user-images.githubusercontent.com/10481028/97904549-6d4ebb00-1d1f-11eb-9fdf-eae0bfbcc18e.png)

![image](https://user-images.githubusercontent.com/10481028/97904602-80618b00-1d1f-11eb-988b-cb19c7eb86f6.png)


Resolves #543 

## Relevant technical choices


I've decided to write the function that calculates the average rating based on an array of reviews at the global helper file so it's easier to find if we need to get this data somewhere else.

## QA Steps


1. Open the details of a technology that has no reviews
2. Verify that avarage doesn't show up
3. Open the details of a technology that has reviews
4. Verify that the average is shown and the information is correct

## Checklist
